docs/data/material/getting-started/templates/landing-page/getLPTheme.tsx

```diff
@@ -0,0 +1,376 @@
+          },
+          warning: {
+            main: '#F7B538',
+            dark: '#F79F00',
+          },
+          error: {
+            light: red[50],
+            main: red[500],
+            dark: red[700],
+          },
+          success: {
+            light: successColor[300],
+            main: successColor[400],
+            dark: successColor[800],
+          },
+          background: {
+            default: '#fff',
+            paper: greyColor[50],
+          },
+          text: {
+            primary: greyColor[800],
+            secondary: greyColor[600],
+          },
+        }
+      : {
+          primary: {
+            light: brandColor[400],
+            main: brandColor[500],
+            dark: brandColor[800],
+          },
+          secondary: {
+            light: secondaryColor[400],
+            main: secondaryColor[500],
+            dark: secondaryColor[900],
+          },
+          warning: {
+            main: '#F7B538',
+            dark: '#F79F00',
+          },
+          error: {
+            light: '#D32F2F',
+            main: '#D32F2F',
+            dark: '#B22A2A',
+          },
+          success: {
+            light: successColor[400],
+            main: successColor[500],
+            dark: successColor[700],
+          },
+          background: {
+            default: greyColor[900],
+            paper: greyColor[800],
+          },
+          text: {
+            primary: '#fff',
+            secondary: greyColor[300],
+          },
+        }),
+  },
+});
+
+const typographyBase = {
+  fontFamily: "'Inter', sans-serif",
+  fontSize: 14,
+  fontWeightLight: 300,
+  fontWeightRegular: 400,
+  fontWeightMedium: 500,
+};
+
+export default function getLPTheme(mode: 'light' | 'dark') {
+  const modeDesignTokens = getDesignTokens(mode);
+
+  const fontHeader = {
+    color: modeDesignTokens.palette.text.primary,
+    fontWeight: typographyBase.fontWeightMedium,
+    fontFamily: "'Inter', sans-serif",
+  };
+
+  return {
+    ...modeDesignTokens,
+    palette: {
+      ...modeDesignTokens.palette,
+      background: {
+        ...modeDesignTokens.palette.background,
+      },
+    },
+    components: {
+      MuiMenuItem: {
+        styleOverrides: {
+          root: {
+            color: mode === 'light' ? greyColor[500] : greyColor[300],
+            borderRadius: '8px',
+            fontWeight: 500,
+          },
+        },
+      },
+      MuiPaper: {
+        styleOverrides: {
+          root: {
+            backgroundColor: mode === 'light' ? greyColor[100] : greyColor[900],
+          },
+        },
+      },
+      MuiCard: {
+        styleOverrides: {
+          root: {
+            backgroundColor: mode === 'light' ? greyColor[50] : greyColor[900],
+            boxShadow: 'none',
+            border:
+              mode === 'light'
+                ? `1px solid ${alpha(greyColor[200], 0.8)}`
+                : `1px solid ${alpha(greyColor[700], 0.6)}`,
+            borderRadius: '8px',
```

Comment:
To support light and dark mode in zero-runtime, we have added a utility method to theme called `theme.applyDarkStyles()`. You'll have to use that to apply dark mode styles as `mode === "light/dark"` check won't work with zero runtime. The equivalent transformation would look like - 

```js
root: {
  boxShadow: 'none',
  borderRadius: '8px',
  backgroundColor: greyColor[50],
  border: `1px solid ${alpha(greyColor[200], 0.8)}`,
  ...theme.applyDarkStyles({
    backgroundColor: greyColor[900],
    border: `1px solid ${alpha(greyColor[700], 0.6)}`,
  })
}
```

Reply:
Same for other slots.

Reply:
Great! Thanks for this! I'll fix it everywhere once I finish the designs 🙌  

Also, is there anything else I should change for the zero-runtime? I think this is the perfect opportunity to teach users about it :)